src/github.com/travis-ci/jupiter-brain/server/database.go

```diff
@@ -0,0 +1,70 @@
+package server
+
+import (
+	"time"
+
+	// Register PostgreSQL driver bits
+	_ "github.com/lib/pq"
+
+	"github.com/jmoiron/sqlx"
+	"github.com/travis-ci/jupiter-brain"
+)
+
+type database interface {
+	SaveInstance(*jupiterbrain.Instance) error
+	DestroyInstance(string) error
+	FetchInstances(*databaseQuery) ([]*jupiterbrain.Instance, error)
+}
+
+type databaseQuery struct {
+	MinAge time.Duration
+}
+
+type pgDatabase struct {
+	conn *sqlx.DB
+}
+
+func newPGDatabase(url string) (*pgDatabase, error) {
+	conn, err := sqlx.Open("postgres", url)
+	if err != nil {
+		return nil, err
+	}
+
+	return &pgDatabase{
+		conn: conn,
+	}, nil
+}
+
+func (db *pgDatabase) SaveInstance(inst *jupiterbrain.Instance) error {
+	instanceID := ""
```

Comment:
It doesn't look like this is used except for the `Scan` below, so I think it can be deleted?